Bluepill-runner/Bluepill-runner/BPPacker.m

```diff
@@ -85,4 +88,73 @@ + (NSMutableArray *)packTests:(NSArray *)xcTestFiles
     return bundles;
 }
 
+#pragma mark - Private helper methods
+
+/*!
+
+ @brief Updates the config to expand any testsuites in the tests-to-run/skip into their individual test cases.
+ 
+ @discussion Bluepill supports passing in just the 'testsuite' as one of the tests to 'include' or 'exclude'.
+ This method takes such items and expands them out so that the 'packTests:' method above can simply
+ work with a list of fully qualified tests in the format of 'testsuite/testcase'.
+ 
+ @param config the @c BPConfiguration for this bluepill-runner
+ @param xcTestFiles an NSArray of BPXCTestFile's to retrieve the tests from
+ @return an updated @c BPConfiguration with testCasesToSkip and testCasesToRun that have had testsuites fully expanded into a list of 'testsuite/testcases'
+
+ */
++ (BPConfiguration *)normalizeBPConfiguration:(BPConfiguration *)config
+                                withTestFiles:(NSArray *)xcTestFiles {
+    
+    config = [config mutableCopy];
```

Comment:
Since you create a mutable copy the original config is never modified.  SimulatorHelper.testEnvironmentWithConfiguration therefore uses the original config for testCasesToRun which won't work since it doesn't have the suite split out.  Are you able to update this so it will modify the original config?